blocks/library/image/style.scss

```diff
@@ -15,6 +15,11 @@
 		float: right;
 		margin-left: 15px;
 	}
+
+	&[data-align="full"] {
+		width: 90%;
```

Comment:
I think it is trying to go full-width but not too full-width. 100% would probably make more sense though.